variables.tf

```diff
@@ -153,6 +153,18 @@ variable "db_replica_count" {
 
 variable "local_ip" {
   description = "URL used to find user's local IP for use with bastion host"
-  default     = "http://ipv4.icanhazip.com"
+  default     = "http://checkip.amazonaws.com"
```

Comment:
What's the reason for this change?

Reply:
On december and early january the `icanhazip.com` domain was having issues (can't remember if the domain wasn't renewed or the web host wasn't responding), that caused noise when running the terraform on our side.

I haven't had issues with checkip.amazonaws.com ever. Another alternative would be checkip.dns.he.net but that will return your IPv6 address, which could be a gotcha since we expect an IPv4 address returned by this.

Reply:
Makes sense!